### DISCUSSION TITLE
manger status stopped

### DISCUSSION THREAD

**Discussion by wzj1999321** [swar/Swar-Chia-Plot-Manager#746] [Q&A]
not sure why but the manager status stopped

**Comment by mheibert** [comment]
Same Here, manager status stopped after 5 to 7 plots.

`global:
  max_concurrent: 7
  max_for_phase_1: 3
  minimum_minutes_between_jobs: 5

Job:
  - name: IntelNVMe2TB
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: /mnt/ssd1/
    temporary2_directory:
    destination_directory: /mnt/hdd1
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 8000
    max_concurrent: 7
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 45
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 10
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]`

**Comment by Martin-0416** [comment]
Same problem.Hope someone could solve it.

**Comment by mheibert** [comment]
I disabled the swap in Ubuntu and disabled the option in the job "skip disk when full" and disabled prometheus.

Since then the manager keeps running.
